### ISSUE TITLE
Review "OverallCond" feature in future model

### ISSUE THREAD

**Issue by Stefenon** [neo-empresarial/house_pricing_challenge#22]
The "OverallCond" feature shows a different  growth pattern than the others. If the first model doesn't perform as expected, it might be important to review this feature.

**Comment by yvesfracari** [comment]
This issue already be resolved @Stefenon ?

**Comment by Stefenon** [comment]
Not exactly, it seems "OverallCond" influences the model considerably, but its correlation with the target variable doesn't seem to be linear. Perhaps we can close this issue or use some time to try to solve it, but I'm not sure the second option will bring us relevant results.
